examples/util/getroomcredentials.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+/**
+ * Get the Room credentials from the server.
+ * @returns {Promise<{identity: string, token: string}>}
+ */
+function getRoomCredentials() {
+  return fetch('/token').then(function(response) {
+    return response.json();
+  });
```

Comment:
Can use fat-arrow functions and `async`/`await`. Basically, I would like us to use all the modern syntax since it is widely available now, e.g.

```
async function getRoomCredentials() {
  const response = await fetch('/token')
  return response.json();
}
```